### PR TITLE
Simplify classification custom loop examples

### PR DESCRIPTION
**Pull request by yuyu2172**
1. I deleted unused arguments
2. I deleted lines that turn input arrays to variables. In the training loop examples, the inputs to networks are arrays. The current style is inconsistent.

### CODE REVIEW

File: examples/mnist/train_mnist_custom_loop.py [hunk 1]
```diff
@@ -28,8 +28,6 @@ def main():
                         help='GPU ID (negative value indicates CPU)')
     parser.add_argument('--out', '-o', default='result',
```
Comment (toslunar) [inline examples/mnist/train_mnist_custom_loop.py]: The argument `--out` doesn't seem to be used.

**Comment by Crissman** [comment]
Merge conflict fixed. LGTM.

**Comment by okuta** [comment]
@yuyu2172 Sorry, late reply. Cloud you resolve conflict?

**Comment by yuyu2172** [comment]
I think this PR is outdated, since the examples now use `out` and `resume`.
Thus, I would like to close this.

**Comment by okuta** [comment]
OK.

**Pull request closed without merging**